### PR TITLE
Updates to O365 ABAC, Intune Apps, Sensitivity labels

### PR DESCRIPTION
**Pull request by oobedan**
Teams ABACs, including guest and external access
Add PowerShell code snippets (get/set) where available
Supply PowerShell scripts for CMTrace #54 , DefenderATP.ps1 #50 
Redo sensitivity labels #51 
Updated breadcrumbs for portal locations
Other minor O365 ABAC changes

### CODE REVIEW

File: blueprint/abac/intune-applications.md [hunk 1]
```diff
@@ -117,7 +119,9 @@ The following Microsoft Endpoint Manager - Intune (Intune) applications can be f
   * Logo: -
 * Program
   * Install command: `%SystemRoot%\sysnative\WindowsPowerShell\v1.0\powershell.exe -executionpolicy bypass -file DefenderATP.ps1`
+    * The required `DefenderATP.ps1`  can be downloaded here [DefenderATP.ps1](/assets/files/abac/DefenderATP.txt)
   * Uninstall command: `%SystemRoot%\sysnative\WindowsPowerShell\v1.0\powershell.exe -executionpolicy bypass -file DefenderATP.ps1`
+    * The required `DefenderATP.ps1`  can be downloaded here [DefenderATP.ps1](/assets/files/abac/DefenderATP.txt)
```
Comment (zedzedzed) [inline blueprint/abac/intune-applications.md:127]: Is there a separate uninstall script or is it the same as the install script?

These two files should be grouped together similar to https://github.com/govau/desktop.gov.au/pull/69/commits/03b96c10bf7fc9f9fec51c7122613f44e2529124

File: blueprint/abac/office-365.md [hunk 2]
```diff
@@ -625,44 +673,47 @@ Please note, if a setting is not mentioned in the below, it should be assumed to
 
 Table 15 describes the Teams Client configuration settings.
 
+`Microsoft Teams Admin center > Ord-wide settings > Teams Settings `
```
[the remark this answers is not in the document]
Comment (oobedan) [reply blueprint/abac/office-365.md:676]: yes sorry, should be Org-wide

File: blueprint/abac/office-365.md [hunk 3]
```diff
@@ -1118,423 +1210,149 @@ Turn on preservation lock | No
...
-            "Key": "aiptemplatescopes",
-            "Value": "[\"All\"]"
-        }
-    ]
-}
-```
-* Conditions: Not configured
-* Local Settings: Not configured
-* Tooltip: `No damage. This information does not form part of official duty`
+Table 25 lists the sensitivity label policy configuration for Unofficial.
```
[the remark this answers is not in the document]
Comment (oobedan) [reply blueprint/abac/office-365.md:1219]: Re-presented as a table as the json snippet was unusable for the user in current form (was PowerShell output from a Get-), the table is all information required doing the operation from the admin console. 

They could be added to automation piece for later update.

File: blueprint/abac/intune-applications.md (hunk 1 above)
Comment (oobedan) [reply blueprint/abac/intune-applications.md:127]: same script for install/remove currently, uninstall/revert not recommended as they are required security settings

Comment (zedzedzed) [reply blueprint/abac/intune-applications.md]: 👍

**Review by zedzedzed — APPROVED** [review]
Thank you!

**Pull request merged**